packages/hw-app-eth/src/nfts.ts

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+
+type NftInfo = {
+  contractAddress: string;
+  collectionName: string;
+  data: Buffer;
+};
+
+type BackendResponse = {
+  payload: string;
+};
+
+const BACKEND_URL = "https://nft.staging.aws.ledger.fr/v1/ethereum";
```

Comment:
- we need to dev a way to change that URL programmatically/env (i can take it)
- we need a stable endpoint